opa/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+{
+    'name': "Open Academy",
+    'summary': """
+        Manage your academy.""",
+    'author': "Vauxoo, Antonio Aguilar <antonio@vauxoo.com>",
```

Comment:
It's ok for this module add you as author, but for new modules that you create, the only author it's the company
```suggestion
    'author': "Vauxoo".
```

Also, use `'` or `"` on your code, not twice for single strings

Reply:
Done, I removed myself from the authors.